src/components/Attachments/AttachmentView/AttachmentViewImage/index.js

```diff
@@ -28,7 +28,7 @@ function AttachmentViewImage({source, file, isAuthTokenRequired, loadComplete, o
             onPress={onPress}
             disabled={loadComplete}
             style={[styles.flex1, styles.flexRow, styles.alignSelfStretch]}
-            role={CONST.ACCESSIBILITY_ROLE.IMAGEBUTTON}
+            accessibilityRole={CONST.ACCESSIBILITY_ROLE.IMAGEBUTTON}
```

Comment:
Wouldn't this throw a deprecation warning? If so, we should either not use it or remove the `@deprecated` comment from the `const` file where this is defined.

Reply:
I covered this in the description:

> Note: In the IMAGEBUTTON Case, I thought it was best to preserve the imagebutton role information even though it uses the deprecated ACCESSIBILITY_ROLE. This minimizes the number of changes, rather than try to decide on a case-by-case basis what an appropriate replacement would be. ACCESSIBILITY_ROLE.IMAGEBUTTON is marked as deprecated and so it will be flagged and can be cleaned up the next time someone touches that component. In the meantime, the accessibilityRole prop is still supported in react-native (and will be for a long time) so there shouldn't be any problems.

But if you have enough context for this component to suggest what the correct value should be then I can make the change here.